lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -131,12 +131,21 @@ class Output
         }                                                                     \
     } while (false)
 
+#define UNWRAP_RESOURCE(resource, innerType) reinterpret_cast<PlatformAgnosticIntlObject<innerType> *>(resource)->GetInstance();
+
 namespace PlatformAgnostic
 {
 namespace Intl
 {
     using namespace PlatformAgnostic::Resource;
 
+    // This file uses dynamic_cast and RTTI
+    // While most of ChakraCore has left this feature disabled for a number of reasons, ICU uses it heavily internally
```

Comment:
Is there a plan to enable RTTI for ChakraCore broadly at some point? I wonder if it's worth adding a TODO somewhere in `PlatformAgnostic` somewhere to make sure we remove the explicit enablement here at that point.

Reply:
@Penguinwizzard felt very strongly about not (and, I think, never) doing so.

Reply:
I think the sentiment there is pretty strong. There's a binary size cost and runtime cost that make it a non-workable option for Chakra probably. No reason to do it in ChakraCore if we're not going to enable it for Chakra.